docs/wiki/framework/gunbag-framework.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: wiki
+title: Gunbag Framework
+description: Explains how to set-up gunbags
```

Comment:
```suggestion
description: Explains how to set up gunbags
```

Reply:
I copied that from grenades. Might also wanna change there?